packages/clickable/style.ts

```diff
@@ -1,10 +1,7 @@
 import { css } from "emotion";
-import { coreColors } from "../shared/styles/color";
 
-const { white } = coreColors();
 export const outline = css`
   &:focus {
-    box-shadow: 0 0 0 1px ${white}, 0 0 0 3px currentColor;
     outline: 0;
```

Comment:
@mperrotti I decided to leave it here and decide later what to  do about the `outline` because of design wise we do not want to show but a11y I would yes

Reply:
Sounds good. Have you spoken with the design team about how they want to handle this styling? Or should I?